immap.go

```diff
@@ -10,8 +10,8 @@ type IMmap interface {
 	Lock() error
 	Unlock() error
 	Advise(advice int) error
-	ReadUint64At(offset int64) (uint64, error)
-	WriteUint64At(num uint64, offset int64) error
+	ReadUint64At(offset int64) uint64
+	WriteUint64At(num uint64, offset int64)
 	Flush(flags int) error
 	Unmap() error
```

Comment:
Umap is still returning error?

Reply:
Errors returned by Unmap are errors returned by syscall unmap. Not all of those errors are `panic` worthy. We will have to analyze the returned error to be able to confidently choose to panic.

Reply:
When I say `errors returned when region is unmapped` is the error returned by Read* and Write* functions, not Unmap.

Reply:
Ok. Makes sense.